src/shared.js

```diff
@@ -869,16 +871,16 @@ export const searchSpecimenInstitute = async () => {
  * @returns {object} returns a response object
  * 
  */
-export const searchSpecimenByRequestedSite = async (requestedSite) => {
-    logAPICallStartDev('searchSpecimenByRequestedSite');
+export const searchSpecimenByRequestedSiteAndBoxId = async (requestedSite, boxId) => {
```

Comment:
NIT: Should we update the DocBlock above.

Reply:
Thanks for pointing this out.